ui.af.apps/src/main/content/jcr_root/apps/core/fd/components/form/container/v2/container/clientlibs/site/js/formcontainerview.js

```diff
@@ -83,6 +83,10 @@
             // before initializing the form container load all the locale specific json resources
             // for runtime
             const formLanguage = formContainer.getLang();
+            if (formLanguage) {
+                const localeObj = new Intl.Locale(formLanguage);
+                document.getElementsByTagName("body")[0].dir = localeObj.textInfo.direction;
```

Comment:
This should be on the the current form context, since there can be multiple forms, you could use _element and set direction